src/loaders/ImageLoader.js

```diff
@@ -54,23 +54,58 @@ class ImageLoader extends Loader {
 
 			removeEventListeners();
 
-			if ( onError ) onError( event );
+			if ( onError ) {
+
+				if ( this.abortSignal && this.abortSignal.aborted ) {
+
+					// Simulate an error similar to the DOMException thrown by the Fetch API
+					// (DOMException is not instanciable)
+					const e = new Error();
+					e.name = 'AbortError';
+					e.message = 'The operation was aborted.';
+					onError( e );
+
+				} else {
+
+					onError( event );
+
+				}
+
+			}
 
 			scope.manager.itemError( url );
 			scope.manager.itemEnd( url );
 
 		}
 
+		function onAbortSignal() {
+
+			image.src = null;
```

Comment:
Setting image.src to `null` gets interpreted as a relative url. This should be set to an empty string, instead.

```js
const img = new Image();
img.src = null;
console.log( img.src )
// https://threejs.org/docs/null
// 404
```

Reply:
@gkjohnson done, I tested is aborts the loading as expected with an `AbortError` error